packages/ckeditor5-ui/src/bindings/addKeyboardHandlingForGrid.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * @license Copyright (c) 2003-2022, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module ui/bindings/addKeyboardHandlingForGrid
+ */
+
+export default function addKeyboardHandlingForGrid( keystrokes, focusTracker, gridElementsCollection, numberOfColumns ) {
+	keystrokes.set( 'arrowright', evt => {
+		const gridElements = [ ...gridElementsCollection ];
```

Comment:
Why do you convert collection to array here? The `Collection` class has `get( index )` and `getIndex()` and `length`. Is there anything missing? This could have an unnecessary performance implication in large collections.